### PR TITLE
Feedback from Alison, Jeff, and Alicia

### PR DESCRIPTION
**Pull request by aappling-usgs**

### CODE REVIEW

**Comment by aappling-usgs** [comment]
@jread-usgs - I think this course does a great job of laying the foundation. I like the PGDL code example - real yet simple!

I've made a bunch of edits for looks and clarity, and I've addressed all those issues that I think need addressing from what Alicia and Jeff reported (all of which are now documented as Issues in this repo).

I only made one functional change, which was to add one more comment to the PR that the learner creates with the initial remake.yml. We should probably test to make sure I didn't break that step.

**Comment by aappling-usgs** [comment]
I also tried out smee.io - I didn't need to do anything with it, but I see why [they recommend it](https://lab.github.com/docs/testing) for building/debugging complex courses!

image

**Comment by jordansread** [comment]
yes, I was using this too. But I had to set it up each time when I re-started the course (because I was deleting the repo)

Wish it would let us see the data.tree contents. I couldn't find what files were committed in there

File: responses/10-structure.md [hunk 1]
```diff
@@ -1,85 +1,10 @@
...
-text(2.3, 1.1, 'Process-Based', pos = 4, cex = 1.1)
-
-dev.off()
-```
-
-This code has some major issues, including that it uses a directory that is specific to a user, it plots to a non-project file location, and the 
-structure of the code makes it hard to figure out what is happening. This simple example is a starting point for understanding the investments we make
-to move towards code that is more reproducible, more shareable, and understandable. Additionally, we want to structure our code and our projects 
-in a way where we can build on top of them as the projects progress. 
+Often we create temporary code or are sent scripts that look like [my_work_R/my_happy_script.R](my_work_R/my_happy_script.R) in this repository. Take a minute to look through that file now.
```
Comment (jordansread) [inline responses/10-structure.md]: Did you want this to link directly on the web? I think we'd need [my_work_R/my_happy_script.R]({{ repoURL }}/blob/master/my_work_R/my_happy_script.R) right?

File: responses/00-overview.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+* Good organization of code scripts helps you quickly find the file you need, whether you or a teammate created it.
 
-Dependency managers such as `remake`, `scipiper`, and `drake` help you reduce the amount of unnecessary runtime as you're developing new parts of a pipeline.
+* Dependency managers such as `remake`, `scipiper`, and `drake` formalize the relationships among the datasets and functions to ensure reproducibility while also minimizing the amount of unnecessary runtime as you're creating or modifying parts of the pipeline.
 
-Assign yourself to this issue to get started.
+:keyboard: Activity: Assign yourself to this issue to get started.
+
+:bulb: Tip: Throughout this course, I, the Learning Lab Bot, will reply and direct you to the next step each time you complete an activity. But sometimes it takes me 5-30 seconds :hourglass_flowing_sand: to give you a reply, and occasionally you'll need to refresh the current GitHub page to see my reply. Please be patient, and let my humans know (`jread-usgs` or `aappling-usgs`) if I seem to have become completely stuck.
```
Comment (jordansread) [inline responses/00-overview.md:9]: :+1:

**Review by jordansread — APPROVED** [review]

File: responses/10-structure.md (hunk 1 above)
Comment (aappling-usgs) [reply responses/10-structure.md]: Whoops, yes, forgot to figure out what the right link format would be!

**Pull request merged**